docs/credentials.rst

```diff
@@ -101,7 +101,7 @@ Credential Types
 ----------------
 
 .. index::
-   single: credentials; types
+   pair: credentials; types
```

Comment:
Move to a separate PR?

Reply:
No, it is related to putting back the indexes, might as well fix these incorrectly labeled entries instead of another PR.

Reply:
It's the only change in the file. Seems odd to include credentials in a seemingly inventory-related patch.